docs/getting_started/java/hello_world_in_java/index.md

```diff
@@ -55,7 +55,7 @@ Before starting this tutorial:
 
 - [Set up a local development environment for developing Temporal applications using the Java programming language](/getting_started/java/dev_environment/index.md)
 - Follow the tutorial [Run your first Temporal application with the Java SDK](/getting_started/java/first_program_in_java/index.md) to gain a better understanding of what Temporal is and how its components fit together.
-- Ensure a build tool like [Gradle](https://gradle.org/install/) or [Maven](https://maven.apache.org/install.html) is installed and ready to use to create a Java project.
+- Ensure a build tool like [Maven](https://maven.apache.org/install.html) is installed and ready to use to create a Java project.
```

Comment:
Since we're removing Gradle support we can be more forceful here. 
```suggestion
- Ensure the build tool [Maven](https://maven.apache.org/install.html) is installed and ready to use to create a Java project.
```